articles/communication-services/concepts/ui-library/includes/mobile-ui-use-cases.md

```diff
@@ -75,9 +75,9 @@ Localization is key to making products for users around the world and who speak
 
 Accessibility is a key focus of the call libraries. You can use a screen reader to make important announcements about call status and to help ensure that visually impaired users can effectively participate when they use the application.
 
-## Participant view data injection
+## Customization
```

Comment:
I believe "Customization" is not the right word, because is more used to talk about customization colors, fonts ,etc.,  And we want to talk about participants avatars, display name, etc.

Reply:
thank you I will update this